Exrop.py

```diff
@@ -5,18 +5,28 @@
 import code
 
 def parseRopGadget(filename, opt=""):
-    cmd = 'ROPgadget {} --binary {} --multibr --only "pop|xchg|add|sub|xor|mov|ret|jmp|call|syscall|leave" --dump | tail -n +3 | head -n -2'.format(opt, filename)
-    with popen(cmd) as fp:
-        sample_gadgets = dict()
-        datas = fp.read().strip().split("\n")
-        datas.sort(key=len) # sort by length
-        for data in datas:
-            addr,insns = data.split(" : ")
-            insstr,opcode_hex = insns.split(" // ")
-            opcode = bytes.fromhex(opcode_hex)
-            addr = int(addr, 16)
+    from subprocess import Popen, PIPE, STDOUT
+    import re
+
+    cmd = ['ROPgadget', '--binary', filename, '--multibr', '--only',
```

Comment:
this changes breaks the feature where the user can add additional ropgadget options (i will document it later), please add `opt` variable before `--binary`

Reply:
It's been added on two lines later.